### DISCUSSION TITLE
Payment processors that offer their own integration for BoxBilling

### DISCUSSION THREAD

**Discussion by BelleNottelling** [boxbilling/boxbilling#959] [General]
I've found a couple payment processors that have their own integration developed for BoxBilling. Should we link these somewhere or include them in BB out of the box?

- Paygol - Latin America. This one is already in our repository

  - https://github.com/paygol/BoxBilling
- Paymentwall - Looks to be global. This one is a little dated, but I don't think their API has broken.
  - https://github.com/paymentwall/module-boxbilling
 - Cardinity - For euro markets. Most of their payment adapters are on github, but this one isn't, thankfully the download still works.
   - https://cardinity.com/developers/module/boxbilling
 - Paystack - For Africa. This payment adapter has actually been recently updated
   - https://github.com/PaystackHQ/plugin-boxbilling
 - PayFast - South Africa
   - https://github.com/PayFast/mod-boxbilling/

**Comment by timothygwebb** [comment]
If we can include them out of the box such as internet.bs I would recommend
doing so that way it is an easy install for the client and they don't have
to go to the developer to get the product installed or updated.

**Comment by yagiz-dev** [comment]
No, don't merge them to the core. It's their code, and it's their job to maintain the code. Once we get the extension store working, installing third party adapters will be as simple as having them coming out-of-the-box.

**Reply by timothygwebb** [reply]
@evrifaessa Sounds like a plan thank you for your input. :)
